app/core/controllers/users.js

```diff
@@ -15,18 +19,20 @@ module.exports = {
     if (req.params.user) {
       return res.send(await m.user.findById(req.params.user));
     }
-
     return res.render('users', {
       section: 'Users',
-      users: await m.user.find({ 'meta.status': 'active' })
+      users: await m.user.find({ 'meta.status': 'active' }),
+      cities: await m.city.find({ 'meta.status': 'active' })
     });
   },
 
   delete: async (req, res, next) => {
     try {
-      const user = await m.user.findById(req.params.user).exec();
+      const user = await m.user.findById(req.params.user)
+        .exec();
       await user.remove();
-      return res.status(200).end();
+      return res.status(200)
```

Comment:
Response for a `DELETE` method should be consistent throughout the API. In this case the response is not the same as for `DELETE /users`.

Reply:
I did not change anything significant here,  the editor optimized the markup of the code
return res.status(200)
        .end();
Only in app/core/controllers/users.js we are adding in request string 'success' , so now I'll add it here (in app/core/controllers/users.js).